### PR TITLE
Added attestation for julia

### PR DESCRIPTION
**Pull request by LaurentGoderre**

### CODE REVIEW

File: apply-templates.sh [hunk 1]
```diff
@@ -11,6 +11,14 @@ elif [ "$BASH_SOURCE" -nt "$jqt" ]; then
 	wget -qO "$jqt" 'https://github.com/docker-library/bashbrew/raw/9f6a35772ac863a0241f147c820354e4008edf38/scripts/jq-template.awk'
 fi
 
+jqf='.template-helper-functions.jq'
+if [ -n "${BASHBREW_SCRIPTS:-}" ]; then
+	jqf="$BASHBREW_SCRIPTS/template-helper-functions.jq"
+elif [ "$BASH_SOURCE" -nt "$jqf" ]; then
+	wget -qO "$jqf" 'https://github.com/docker-library/bashbrew/raw/master/scripts/template-helper-functions.jq'
```
Comment (yosifkit) [inline apply-templates.sh:18]: The download of `.template-helper-functions.jq` should use a specific commit like the awk script does. This'll help each repo control when it brings in improvements/changes.

File: apply-templates.sh [hunk 2]
```diff
@@ -11,6 +11,14 @@ elif [ "$BASH_SOURCE" -nt "$jqt" ]; then
...
+fi
+
+
```
Comment (yosifkit) [inline apply-templates.sh:21]:
```suggestion
```
Extra empty line.

File: Dockerfile-linux.template [hunk 3]
```diff
@@ -121,7 +122,22 @@ RUN set -eux; \
...
+			params: {
+				os_name: (if is_alpine then "alpine" else "debian" end),
+				os_version: env.variant | ltrimstr("alpine")
+			},
+			licenses: [
+				"MIT"
+			]
+		} | sbom | tostring | @sh
+	}} > $JULIA_PATH/julia.spdx.json;
+
```
Comment (yosifkit) [inline Dockerfile-linux.template:140]:
```suggestion
```
Extra empty line.

**Review by yosifkit — CHANGES_REQUESTED** [review]

**Comment by LaurentGoderre** [comment]
Done

**Review by yosifkit — APPROVED** [review]

**Pull request merged**